docs/sources/mobile-app/push-notifications/index.md

```diff
@@ -25,28 +25,37 @@ To receive push notifications from the Grafana OnCall mobile app, you must add t
 In the **Settings** tab of the mobile app, tap on **Notification policies** to review, reorder, remove, add or change steps.  
 Alternatively, you can do the same on desktop. From Grafana OnCall, navigate to the **Users** page, click **View my profile** and navigate to the **User Info** tab.
 
-<img src="/static/img/oncall/mobile-app-notification-policies.png" width="300px">
+<img src="/static/img/oncall/mobile-app-v1-android-notification-policies.png" width="300px">
 
 ## Configuration
 
 Use the **Push notifications** section in the **Settings** tab to configure push notifications.
 
 You can always confirm how a notification is presented by going to Grafana OnCall on your desktop,
 navigate to the **Users** page, click **View my profile** and navigate to the **Mobile App connection** tab.
-Here you can send a test notification of default or important priority.
+Here you can send a test notification of default or important priority.  We recommend doing this to try out
+correct configuration of **Do Not Disturb** and **Volume** overrides.
 
 ### Android
 
 On Android, we leverage the "Notification channels" system feature.
 Each type of notification (**important**, **default**, and **on-call shifts**) registers a channel.
-In this channel, you may configure the sound style, optional Do Not Disturb override, vibration, and so on.
+In this channel, you may configure the sound style, vibration, and so on.
 **Customize notifications** takes you to this system menu, while hitting the **back** button or swiping left (if enabled) takes you back to the application.
 
+>**Note**: You can explore included sounds and recommendations via the **Sound Library** button, but to change the sound, go to **Customize notifications**.
+
+#### Override Do Not Disturb
+
+- On standard Android versions, the **Override Do Not Disturb** option is available in the channel options described above.
+- On Samsung devices, you can add the Grafana Oncall app under (System) Settings > Notifications > Do not disturb > App notifications.
```

Comment:
What do you think about adding something like
"On some Android devices there might be a non-standard DND system settings. We invite you to check in your smartphone settings if you can't find it through the app settings."
Something like this, but with better wording

Reply:
you're right that i think we should address the case of not-stock and not-samsung, even if it's only a small group. i will rework this.